data/json/recipes/food/brewing.json

```diff
@@ -522,5 +522,27 @@
     "qualities": [ { "id": "BOIL", "level": 2 } ],
     "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
     "components": [ [ [ "milk_standard_raw", 9, "LIST" ] ], [ [ "cabbage", 1 ], [ "chili_pepper", 1 ], [ "yoghurt", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "brew_rootbeer",
+    "result_mult": 7,
```

Comment:
I'm not super well versed in json recipe tags, but does this mean the recipe produces 7 times the output? Is this really correct?

Reply:
I also was unsure about this tag and so I looked at other recipes, I think you are correct.

Reply:
It definitly produced 7x output, will push change to make it less so.